server/routes/index.js

```diff
@@ -1,7 +1,9 @@
 import userRouter from './users';
 import articleRouter from './article';
+import socialLoginRouter from './socialLogin';
 
 export default (app) => {
   app.use('/api/v1', userRouter);
   app.use('/api/v1/articles', articleRouter);
+  app.use('/api/v1', socialLoginRouter);
```

Comment:
Priscilla has another implementation of this. Please revisit and refactor accordingly.

Reply:
I'll fix this when I'm resolving merge conflicts.